src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -0,0 +1,98 @@
+        return dtoMapper.toDto(productService.add(dtoMapper.toModel(productRequestDto)));
+    }
+
+    @GetMapping("/{id}")
+    @ApiOperation(value = "Get product by id")
+    public ProductResponseDto findById(@PathVariable Long id) {
+        return dtoMapper.toDto(productService.get(id));
+    }
+
+    @DeleteMapping("/{id}")
+    @ApiOperation(value = "Delete product from database by id")
+    public void delete(@PathVariable Long id) {
+        productService.delete(id);
+    }
+
+    @PutMapping("/{id}")
+    @ApiOperation(value = "Update product in database by id")
+    public ProductResponseDto update(@PathVariable Long id,
+                                     @RequestParam ProductRequestDto requestDto) {
+        Product product = dtoMapper.toModel(requestDto);
+        product.setId(id);
+        return dtoMapper.toDto(productService.update(product));
+    }
+
+    @GetMapping("/find-all")
+    @ApiOperation(value = "Get all products")
+    public List<ProductResponseDto> findAll(@RequestParam(defaultValue = "20")
+                                                @ApiParam(value = "default value is 20")
+                                                Integer count,
+                                            @RequestParam(defaultValue = "0")
+                                                @ApiParam(value = "default value is 0")
+                                                Integer page,
+                                            @RequestParam(defaultValue = "id")
+                                                @ApiParam(value = "default value is id")
+                                                String sortBy) {
+        Sort sort = Sort.by(ProductSortUtilParser.toSortOrders(sortBy));
+        PageRequest pageRequest = PageRequest.of(page, count, sort);
+        return productService.findAll(pageRequest).stream()
+                .map(dtoMapper::toDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/by-price")
+    @ApiOperation(value = "Get all product between price")
+    public List<ProductResponseDto> findAllByPrice(@RequestParam BigDecimal from,
+                                                   @RequestParam BigDecimal to,
+                                                   @RequestParam(defaultValue = "20")
+                                                        @ApiParam(value = "default value is 20")
+                                                        Integer count,
+                                                   @RequestParam(defaultValue = "0")
+                                                        @ApiParam(value = "default value is 0")
+                                                        Integer page,
+                                                   @RequestParam(defaultValue = "id")
+                                                        @ApiParam(value = "default value is id")
+                                                        String sortBy) {
```

Comment:
let's describe more what these fields are used for, and not just their default value, and also not forget about `from` and `too`